tests/cypress/latest/e2e/unit_tests/first_connection.spec.ts

```diff
@@ -0,0 +1,26 @@
+/*
+Copyright © 2023 - 2024 SUSE LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import * as cypressLib from '@rancher-ecp-qa/cypress-library';
+import { qase } from 'cypress-qase-reporter/dist/mocha';
+
+
+Cypress.config();
+describe('First login on Rancher', () => {
+  qase(46,
```

Comment:
@cpinjani this QASE is not present in Fleet project, can you please explain from where it is located?

Reply:
Good point, its a placeholder and doesn't corresponds to QASE number as of now.
Can be changed it to matching one (post QASE integration with automation runs)

Reply:
For pose QASE integration, do we need to add those test cases in the Fleet project ?

Reply:
They can be added if missing

Reply:
Okay, thanks @cpinjani :smile: